File: redisHelloWorld.py
#!/usr/bin/python
from redis import StrictRedis
import sys

hostname=sys.argv[1]
password=sys.argv[2]

r = StrictRedis(host=hostname,port=6379,password=password,db=0)

def getPacktWelcome():
    #GET value stored in packt:welcome
    print("Displaying current welcome message...")
    value = r.get('packt:welcome')
    print("message = " + str(value))

def setPacktWelcome():
    #SET new value packt:welcome
    print("Writing \"Hello world from Python!\" to Redis...")
    r.set('packt:welcome','Hello world from Python!')

getPacktWelcome()
setPacktWelcome()
getPacktWelcome()

File: redis/RedisHelloWorld.java
package redisQueryUser;

public class RedisHelloWorld {

	public static void main(String[] args) {
		RedisConnection conn = new RedisConnection("127.0.0.1","currentHorseBatteryStaple");
	    System.out.println("Connected to Redis");

	    String key = "packt:welcome";
	    String newMessage = "Hello world from Java!";
	    
	    //GET value stored in packt:welcome
	    System.out.println("Displaying current welcome message...");
//	    String message = conn.get("packt:welcome");
	    
	    //SET new value packt:welcome
	    String message = conn.getSet(key,newMessage);
	    System.out.println(message);
	    System.out.println("Writing \"" + newMessage + "\" to Redis...");
	    
	    //GET value stored in packt:welcome
	    System.out.println("Displaying the new welcome message...");
	    message = conn.get("packt:welcome");
	    System.out.println(message);
	    
	    conn.close();
	}

}
